web/client/themes/default/less/common.less

```diff
@@ -1,3 +1,7 @@
+#container {
+    overflow: hidden;
```

Comment:
Too generic. This set overflow: hidden to every #container HTML element. That's very dangerous. Is there any other solution?